### PR TITLE
Initial draft of post number 42

### PR DESCRIPTION
**Pull request by jonallured**
Did a draft of this during writing office hours today, any feedback for me @pepopowitz?

### CODE REVIEW

File: source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+favorite: true
+id: 42
+title: Start and Stop Your Work Day With a List
+---
+
+I've developed a habit that's at the heart of my personal operating system: I
```
Comment (pepopowitz) [inline source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md:7]: "my personal operating system" ❤️ ❤️ ❤️

File: source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+* open dashboards
+* add meetings here
+* say hello in Slack
+* read through Slack
+* process email
+* review GH notifications
+* review pulls
+* review Jira
+
+This list is what I follow to begin my day and it goes from least distracting to
```
Comment (pepopowitz) [inline source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md:30]: totally non-blocking - distracting has a pretty negative connotation - do you mean "shallowest to deepest" or maybe something else?

File: source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+## Is There an MVP of This Habit?
+
+If you want to start building a habit like the one I've just described, here's
+what I'd recommend you start with:
+
+Start List
+
+* review calendar: look at the meetings on your calendar - are you prepared for
+  them? Can you cancel any of them?
+* open chat app: wait to open chat until you're ready.
```
Comment (pepopowitz) [inline source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md:84]: 😲

File: source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  meetings by checking for the time of your first meeting the next day.
+
+## What Else Should I Add?
+
+My lists have evolved over time to include hints, prompts, and nudges that I
+find useful - you should be on the look out for these too! An item on these
+lists can be anything but incorporating this basic flow into your life is what
+sets you up to build upon it and that's where the real power comes from.
+
+[Todoist]: https://todoist.com/r/jonathan_allured_mzkvzo
```
Comment (pepopowitz) [inline source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md:103]: is this intended to take you to a user profile or something? It takes me to the todoist home page. 

I suspect this is accidental but just pointing out that it's got some user-id looking thing that might be best not exposed.

**Review by pepopowitz — APPROVED** [review]
❤️ ❤️ ❤️ 

This is great and I think you should ship it.

File: source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md (hunk 4 above)
Comment (jonallured) [reply source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md:103]: Yep, that was a mistake, thanks!! 🤗

File: source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md (hunk 2 above)
Comment (jonallured) [reply source/posts/7070-07-07-start-and-stop-your-work-day-with-a-list.html.md:30]: Yep, this wasn't quite what I meant - I have a rewording that I'm finishing up. 👍

**Pull request merged**